src/intro-gtfs-rt.html

```diff
@@ -0,0 +1,230 @@
+description: >
+  Managed by Cal-ITP, this website seeks to make the transition to contactless
+  fare collection and GTFS Realtime compliance as easy and cost-effective as
+  possible.
+
+show_newsletter: true
+newsletter_background: bg-dark-tan
+---
+
+<div class="container container--skinny">
+  <article>
+    <h1 class="text-center">{{ page.title }}</h1>
+    <p class="text-center fs-7 fw-medium">Everything you need to know</p>
+    <hr />
+    <p class="mb-2">
+      <strong>Where is my bus?</strong> Real-time data is how both riders and operators answer this question. GTFS allows you to
+      see your vehicle—in real time, on a map.
+    </p>
+    <h2 class="h3 pt-4">So, what exactly is GTFS?</h2>
+    <p class="mb-2">
+      GTFS feeds allow public transit agencies to share their transit data, making it easier for developers to create applications
+      that can use this information effectively. Transit service providers produce schedules and prices; third-party applications
+      (like Google Maps, Transit App, etc.) consume this data. Riders use this data to plan for their transit trip.
+    </p>
+    <p class="mb-2">
+      There are two main components: Schedule and Realtime. GTFS Schedule is the plan of what transit service looks like; GTFS
+      Realtime is what is actually happening with the transit service.
+    </p>
+    <div class="row gx-2 my-3">
+      <div class="col-md-6 mb-3 mb-md-0">
+        <div class="rounded rounded-3 border border-info bg-info-subtle p-3 h-100">
+          <h3 class="h4 with-gtfs-schedule d-flex align-items-center">GTFS Schedule</h3>
+          <ul class="mb-0">
+            <li>Static information</li>
+            <li>Describes the routes and schedule of a transit agency</li>
+            <li>Establishes the baseline transit information</li>
+          </ul>
+        </div>
+      </div>
+      <div class="col-md-6">
+        <div class="rounded rounded-3 border border-warning bg-warning-subtle p-3 h-100">
+          <h3 class="h4 with-gtfs-realtime d-flex align-items-center">GTFS Realtime</h3>
+          <ul class="mb-0">
+            <li>Dynamic information</li>
+            <li>Describes the location of vehicles in real time</li>
+            <li>Adjusts transit information based on real-time updates</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <p class="mb-2">
+      GTFS Schedule is the backbone of transit data. GTFS Schedule and GTFS Realtime depend on each other. GTFS Realtime needs
+      GTFS Schedule to give it the baseline operations assumptions. It then enhances this data by adding in real-time data.
+    </p>
+    <p class="mb-2">They each provide comprehensive information in a standardized format.</p>
+    <div class="row gx-2 my-3">
+      <div class="col-md-6 mb-3 mb-md-0">
+        <div class="rounded rounded-3 border border-info bg-info-subtle p-3 h-100">
+          <h3 class="h4 with-gtfs-schedule d-flex align-items-center">GTFS Schedule</h3>
```

Comment:
```suggestion
          <h3 class="h4 with-gtfs-schedule d-flex align-items-center">GTFS Schedule Data</h3>
```

My only remaining concern is the duplicate `h3` headings for the little boxes under the "So, what exactly is GTFS?" heading.

| preview | doc structure |
|-|-|	
| ![image](https://github.com/user-attachments/assets/3e17f001-d75b-4408-866d-89888f810ba7) | ![image](https://github.com/user-attachments/assets/3ebd3c89-276a-4bfd-afd8-376c9cf3f1c6) |

Reply:
If we're going this route (I think it's a good catch and I agree) -- the `GTFS Schedule` heading in the second set of boxes should be updated as well: `GTFS Schedule Data`

Reply:
Hrm... What should the H3 to? Or just.. not use Headings for the GTFS Schedule/GTFS Realtime boxes at all?

Reply:
As long as this heading is changed to include "data," then all the h3 headings will be unique and no further changes are needed.

If you want to move away from using an h3, then we could probably use a `<p>` tag and tie it to the `<ul>` with an `aria-labelledby`. But I think the h3 is fine, it just needs to include "data" so that it's not conflicting with duplicates in the same section

Reply:
Yeah it's the repeated heading text that is a problem, right? So just changing the text of the second heading fixes it.

Reply:
OOOH~~~ I only just noticed the different `Data` name on Figma 😅 😅 

Thanks for the careful eyes and the commit suggestions Vlad! Resolved. Phew!